.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @haikalpribadi @flyingsilverfin
+* @haikalpribadi @flyingsilverfin @lolski
```

Comment:
No, I meant just give `//bin` to @lolski , @alexjpwalker

Reply:
Ahh, should have read the whole message

Reply:
fixed